e2e/cephfs.go

```diff
@@ -102,6 +102,7 @@ func createORDeleteCephfsResources(action kubectlAction) {
 			filename:   cephFSDirPath + cephFSProvisioner,
 			namespace:  cephCSINamespace,
 			oneReplica: true,
+			enableFencing: true,
```

Comment:
how come this isn't `go fmt`'ed?

strange we manage to get through the CI jobs with this :thinking: